packages/google_sign_in/google_sign_in_web/lib/src/flexible_size_html_element_view.dart

```diff
@@ -78,6 +78,8 @@ class _FlexHtmlElementView extends State<FlexHtmlElementView> {
   ) {
     final web.DOMRectReadOnly rect = resizes.toDart.last.contentRect;
     if (rect.width > 0 && rect.height > 0) {
+      // The following ignore can be removed once the package leaves web:0.5.1 behind.
+      //ignore:noop_primitive_operations
```

Comment:
Add a space

`// ignore`

Reply:
Actually there should be space after the colon as well:

`// ignore: noop_primitive_operations`

Reply:
Yep, I used the "quick fix" from vscode, and it told me it should be this:

```
// ignore: noop_primitive_operations
```